config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+# get "users", to: "user#index"
+resources :users,only:[:index,:show]
+get "/users/:user_id/posts", to:"posts#index"
+get "/users/:user_id/posts/:id", to:"posts#show"
```

Comment:
- [ ] [optional]@manzitresor Great work on your routes, I recommend you use resources for all routes as its easier to read and scale in the future.
```suggestion
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
@danmainah  Thank so much for constructive feedback,
I will implement that